packages/zoe/src/zoeService/zoeSeat.js

```diff
@@ -159,7 +160,7 @@ export const makeZoeSeatAdminFactory = baggage => {
           assertHasNotExited(this, 'Cannot exit seat. Seat has already exited');
 
           state.exiting = true;
-          E.when(
+          void E.when(
```

Comment:
As discussed, this solves the problem with the linter, but leaves error messages in production logs that are being noticed. I think we should take care of both with a noop catch clause.

Reply:
@mhofman you tend to be most affected by unhandled promise rejections in the logs. Do you prefer a rejection here to be silent?

Reply:
I don't think I have enough background on the behavior. what are the consequences of `doExit` failing? It seems to not be reported anywhere except in console output. I thought `void`ing a promise was meant to say "I've handled the rejection, trust me".

Reply:
I do not understand what the complaints in the log are unhappy about. This is already in an `E.when()` with an `onRejected` branch.

----

 If the `E.when()` itself fails, and the `onRejected` clauses don't do their job, aren't we in a world of hurt?
 
 @turadg and I talked about this out-of-band, and we concluded that `E.when()` may need a `void` in order to make the linter happy, but a noop catch clause would never prevent messages to the logs. In the future, I'll ask for noop catches only on other kinds of bare promises.